pkg/transforms/batch.go

```diff
@@ -51,7 +51,6 @@ func NewBatchByCount(batchThreshold int) (*BatchConfig, error) {
 		batchThreshold: batchThreshold,
 		batchMode:      BatchByCountOnly, //Default to CountAndTime
 	}
-	config.done = make(chan bool, 1)
```

Comment:
why remove this?

Reply:
Looks like you some how pulled in part of Mike's PR